bin/repair/work_consolidation

```diff
@@ -6,4 +6,4 @@ bin_dir = os.path.split(__file__)[0]
 package_dir = os.path.join(bin_dir, "..", "..")
 sys.path.append(os.path.abspath(package_dir))
 from core.scripts import WorkConsolidationScript
-WorkConsolidationScript(force=True).run()
+WorkConsolidationScript(force=False).run()
```

Comment:
This isn't relevant to the main bug. It's the circulation part of the change I made as part of https://github.com/NYPL-Simplified/server_core/pull/245.